test/plugin/test_out_webhdfs.rb

```diff
@@ -316,4 +316,99 @@ def test_time_key_without_buffer_section
       assert_equal "2017-01-24T20:10:30Z\ttest.now\t{\"message\":\"yay\",\"name\":\"tagomoris\"}\n", line
     end
   end
+
+  sub_test_case "kerberos config" do
+    CONFIG_KERBEROS = config_element(
+      "ROOT", "", {
+        "namenode" => "server.local:14000",
+        "path" => "/hdfs/path/file.%Y%m%d.%H%M.log",
+        "username" => "hdfs_user",
+        "kerberos" => true,
+        "kerberos_keytab" => "/path/to/kerberos.keytab",
+      })
+
+    test "renew_kerberos_delegation_token default" do
+      mock.proxy(WebHDFS::Client).new("server.local", 14000, "hdfs_user", nil, nil, nil, {}, nil).once
+
+      d = create_driver(CONFIG_KERBEROS)
+
+      assert_equal(
+        {
+          kerberos: true,
+          renew_kerberos_delegation_token: false,
+          renew_kerberos_delegation_token_time_hour: nil,
+        },
+        {
+          kerberos: d.instance.kerberos,
+          renew_kerberos_delegation_token: d.instance.instance_eval("@renew_kerberos_delegation_token"),
+          renew_kerberos_delegation_token_time_hour: d.instance.instance_eval("@renew_kerberos_delegation_token_time_hour"),
+        })
+    end
+
+    test "default renew_kerberos_delegation_token_time" do
+      expected_hour = '8h'
+
+      mock.proxy(WebHDFS::Client).new("server.local", 14000, "hdfs_user", nil, nil, nil, {}, expected_hour.to_i).once
+
+      d = create_driver(CONFIG_KERBEROS +
+                        config_element("", "", { "renew_kerberos_delegation_token" => true }))
```

Comment:
@ashie 
Because i do not pass `renew_kerberos_delegation_token_time` thru config_elements, it occur below error.

```
NoMethodError: undefined method `/' for "8h":String
```

or If i pass it, this test code is same with `renew_kerberos_delegation_token_time`
what should i do? give me some advise.

Reply:
Fixed in f1df7fc65a332651a277d2fdcf3416ce1a66fbe5